wpilibj/src/athena/java/edu/wpi/first/wpilibj/CameraServer.java

```diff
@@ -55,8 +54,7 @@ public static CameraServer getInstance() {
   private HashMap<String, VideoSink> m_sinks;
   private HashMap<Integer, ITable> m_tables;  // indexed by source handle
   private ITable m_publishTable;
-  private VideoListener m_videoListener;
-  private int m_tableListener;
+  private VideoListener m_videoListener; //NOPMD
```

Comment:
@PeterJohnson Does this need to be saved or can this field be converted to a local variable?